mongo/datadog_checks/mongo/config_models/validators.py

```diff
@@ -7,4 +7,6 @@
 def initialize_instance(values, **kwargs):
     if 'hosts' not in values and 'server' not in values:
         raise ValidationError('Hosts is a required field')
+    if 'hosts' in values and type(values['hosts']) == str:
```

Comment:
This file holds additional validations and transformations of the config models. It is only used on python 3